configs/milvus.yaml

```diff
@@ -268,9 +268,11 @@ queryNode:
   loadMemoryUsageFactor: 1 # The multiply factor of calculating the memory usage while loading segments
   enableDisk: false # enable querynode load disk index, and search on disk index
   maxDiskUsagePercentage: 95
-  cache:
+  cache: # deprecated, TODO: remove it
     enabled: true
     memoryLimit: 2147483648 # 2 GB, 2 * 1024 *1024 *1024
+  chunkCache:
+    madvice: MADV_WILLNEED # The read ahead policy of chunk cache, options: MADV_NORMAL, MADV_RANDOM, MADV_SEQUENTIAL, MADV_WILLNEED, MADV_DONTNEED
```

Comment:
1. I'd like to move this into cache, the deprecated cache config items could be removed, but keep cache itself
2. Rename this item to 'policy', and the value to 'willneed', we need to decouple the policy and the mmap advice, user may have no experience about mmap, and Milvus may add buffer pool

Reply:
policy may be confused, how about `readAheadPolicy`